hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/flow/dispatch/logic/IsDuplicate.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.handle.flow.dispatch.logic;
+
+/**
+ * Enumerates whether the transaction is a duplicate.
+ */
+public enum IsDuplicate {
+    YES,
+    NO
+}
```

Comment:
Will there ever be anything in between YES or NO? This feels overkill

Reply:
To avoid showing just boolean as true and false in code, this will help readability. That was the thought behind adding this.